user-service/index.js

```diff
@@ -17,11 +17,11 @@ app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 app.use(
   cors({
-    origin: ['http://localhost:3000'],
+    origin: process.env.ENV == 'PROD'? process.env.FRONTEND_URL : 'http://localhost:3000',
     credentials: true,
+    optionsSuccessStatus: 200

```

Comment:
May I know the reason of setting the status code to `200`?

Reply:
Good catch, I used it to test whether this is the cause of issue on production earlier. But I forgot to remove it after fixing the issue. The reason I use it to test was because I thought that some browsers might not support success status 204 (default) and hence revert it to 200 which is supported by older version of browsers.